tests/Microsoft.Identity.Web.Test/FmiTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Identity.Abstractions;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Web.Test.Common.Mocks;
+using Microsoft.Identity.Web.TestOnly;
+using Xunit;
+
+namespace Microsoft.Identity.Web.Test
+{
+    public class FmiTests
+    {
+        [Fact]
+        public async Task FmiPathSentToEndpointTest()
+        {
+            await RunHappyPathTestAsync();
+        }
+
+        private async Task RunHappyPathTestAsync()
+        {
+            // Arrange
+            var tokenAcquirerFactory = InitTokenAcquirerFactoryForFmi();
+            IServiceProvider serviceProvider = tokenAcquirerFactory.Build();
+            var mockHttpClient = serviceProvider.GetRequiredService<IMsalHttpClientFactory>() as MockHttpClientFactory;
+
+            mockHttpClient?.AddMockHandler(MockHttpCreator.CreateHandlerToValidatePostData(
```

Comment:
Use the https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-forgiving operator instead. `mockHttpClient!.AddMockHandler`